runbotics/runbotics-orchestrator-ui/src/src-app/components/ProcessVariablesPanel/VariablesPanel/VariablesPanel.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import {
+    Box, Typography, Chip, Grid, Divider, IconButton
+} from '@mui/material';
+
+import { useTheme } from '@mui/material/styles';
+
+
+import If from '#src-app/components/utils/If';
+import useProcessVariables from '#src-app/hooks/useProcessVariables';
+import { translate } from '#src-app/hooks/useTranslations';
+
+import MenuCopy from '../MenuCopy';
+import { GridContainer, GridTag, GridVariable
+} from './VariablesPanel.styles';
+
+export enum VariableTag {
+    Variable = 'VariableTag',
+    ActionOutput = 'ActionOutputTag',
+}
+
+const VariablesPanel = () => {
+    const theme = useTheme();
+    const { globalVariables, inputActionVariables, outputActionVariables, attendedVariables } = useProcessVariables();
+    const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+    const [showMenu, setShowMenu] = useState(null);    
+    
+    const [menuId, setMenuId] = useState(false);
```

Comment:
Hm, can't we use just one state, something like `openMenuId`? And if there is `openMenuId` just show it

Reply:
also, if we have state like an ID we need to start from null/undefined not false